### ISSUE TITLE
Template/Style not working properly in Python3

### ISSUE THREAD

**Issue by edwardoliveira** [romanlv/trml2pdf#8]
Hello, I have tested the following `report.rml` on Python3:

```
<!DOCTYPE document SYSTEM "rml.dtd">
<document filename="example_2.pdf">
<template>
<pageTemplate id="main">
<frame id="first" x1="72" y1="72" width="451" height="698"/>
</pageTemplate>
</template>
<stylesheet>
</stylesheet>
<!-- The story starts below this comment -->
<story>
<para>
This is the "story". This is the part of the RML document where
your text is placed.
</para>
<para>
It should be enclosed in "para" and "/para" tags to turn it into
paragraphs.
</para>
</story>
</document>
```

I am using Python3 and the script I use is as follows:

```
from trml2pdf import parseString

i = open('report.rml', 'r+')
x = i.read()
y = parseString(x)
i.close()

output = open('report.pdf', 'wb')
output.write(y)
output.close()
```

Well, I run the script above it renders as below. As you can see, the break lines are printed verbatim. 

![screenshot from 2016-01-12 17 20 29](https://cloud.githubusercontent.com/assets/9326037/12274204/ff16dda2-b950-11e5-8b45-2c87466eff11.png)

If I replace the original text by including accents like the word `Resolução` then this word gets printed as `Resolu\xc3\xa7\xc3\xa3o`. Anyone could help me solve this problem?

Thanks in advance!

**Comment by edwardoliveira** [comment]
@romanlv **update:** I have just submitted PR ( https://github.com/romanlv/trml2pdf/pull/9 ) that partially addresses this problem by removing the new line and tab space escape characters, but I don't know if this is the best solution yet. wdyt?

**Comment by romanlv** [comment]
fixed by #10
